internal/key/key.go

```diff
@@ -174,6 +175,21 @@ func SSHSSOPublicKey(ctx context.Context, client runtimeclient.Client) (string,
 	return sshSSOPublicKey, nil
 }
 
+func GetReleaseComponents(ctx context.Context, client runtimeclient.Client, releaseName string) (map[string]string, error) {
```

Comment:
We are using this function from two different commands (`template cluster` and `template nodepool` and for `aws` and `azure`) so I decided to put it on the `key` package.